docs/reference/dev_help/api_standalone.md

```diff
@@ -17,20 +17,31 @@ catalog = {"Description": "This catalog includes fundamental Python type operato
                         [Bool,    []]]}
 ```
 
-A catalog must be then sourced in Shift either by using the *Catalog Manager* from the Shift UI or by adding the path to the folder containing the catalog file to the `SHIFT_CATALOG_PATH` environment variable.
+>[!NOTE]
+> The *Operators* list from the `catalog` is a list of lists containing two elements. The first one corresponds to the operator class, and the second one is a list of compatible hosts for the operator. If no host is specified, the node will be usable in any context. For instance, if the second element is `["maya", "houdini"]` the operator will be only available for those DCC's. Refer to [Integrations & Resources](../../integration_resources/integrations_resources.md) for more information.
+
+There are two ways of adding a custom catalog to Shift:
+
+- **As a User Catalog**: Use the [Catalog Manager](../catalogs/#the-catalog-manager) to source the catalog file and add it to Shift. This catalog is saved in the user preferences.
+- **As a Environment Catalog**: Add the **path to the directory** the containing the catalog file to the `SHIFT_CATALOG_PATH` environment variable. All files in this path containing a `catalog` object will be identified as custom catalogs and loaded by Shift.
```

Comment:
```suggestion
- **As a Environment Catalog**: Add the **path to the directory** containing the catalog file to the `SHIFT_CATALOG_PATH` environment variable. All files in this path containing a `catalog` object will be identified as custom catalogs and loaded by Shift.
```

Reply:
I changed "As a Environment Catalog" to "As an Environment Catalog" in the suggestion